### PR TITLE
#Task 9 : Update footer-section

### PR DESCRIPTION
**Pull request by TheHienDuong**
[#Task9](https://trello.com/c/HZYWlnyi/3-task-9-create-footer-section)

- Create footer-section

### Overview
image

### CODE REVIEW

File: practice-01/index.html [hunk 1]
```diff
@@ -188,7 +195,8 @@ <h5 class="card__title">PRODUCT</h5>
                     </div>
                     <div class="card__body">
                         <div class="card--horizontal-block">
-                            <h3 class="card__text"> Sales presentations: templates, examples and ideas on how to present like a pro</h3>
+                            <h3 class="card__text"> Sales presentations: templates, examples and ideas on how to present
+                                like a pro</h3>
```
Comment (phattruong-agilityio) [inline practice-01/index.html]: Why have line-break here

File: practice-01/index.html [hunk 2]
```diff
@@ -268,13 +281,14 @@ <h4 class="card__footer">27 Dec 2022,<span class="card__footer--gray"> by Roxie
             <div class="cta__content">
                 <div class="cta__passage ">
                     <p class="cta__title font--large  ">Ready to supercharge your business?</p>
-                    <p class="cta__text  font--medium ">Ask about Yoora products, pricing, implementation, or anything
-                        else.<br> Our highly trained reps are
+                    <p class="cta__text  font--medium ">Ask about Yoora products, pricing, implementation, or
+                        anythingelse.<br> Our highly trained reps are
```
Comment (phattruong-agilityio) [inline practice-01/index.html]: Why have line break here

File: practice-01/index.html [hunk 3]
```diff
@@ -292,14 +306,101 @@ <h4 class="card__footer">27 Dec 2022,<span class="card__footer--gray"> by Roxie
...
     footer 
     --------------------->
-    <footer>
-        <div class="container">
-            <div class="footer__content">
-                <div class="content__left"></div>
-                <div class="content__right"></div>
+    <footer class="footer">
+        <div class="footer__content container">
+            <div class="content__left">
```
Comment (phattruong-agilityio) [inline practice-01/index.html]: wrong BEM name

File: practice-01/index.html [hunk 4]
```diff
@@ -292,14 +306,101 @@ <h4 class="card__footer">27 Dec 2022,<span class="card__footer--gray"> by Roxie
...
+                <a href="#" class="logo">
+                    <img src="./img/logo.png" alt="logo">
+                </a>
+                <div class="footer__text">
```
Comment (phattruong-agilityio) [inline practice-01/index.html]: do we need footer__text

File: practice-01/index.html [hunk 5]
```diff
@@ -292,14 +306,101 @@ <h4 class="card__footer">27 Dec 2022,<span class="card__footer--gray"> by Roxie
...
+                        </li>
+                        <li>
+                            <img src="./img/Linkedin.png" alt="Here is a photo of the media above" class="media__icon">
+                        </li>
+                    </ul>
+
+
+                </div>
+            </div>
+            <div class="content__right">
```
Comment (phattruong-agilityio) [inline practice-01/index.html]: wrong BEM name

File: practice-01/index.html [hunk 6]
```diff
@@ -292,14 +306,101 @@ <h4 class="card__footer">27 Dec 2022,<span class="card__footer--gray"> by Roxie
...
+                <div class="footer__list">
+                    <ul class="list__item">
```
Comment (phattruong-agilityio) [inline practice-01/index.html]: wrong BEM name

File: practice-01/index.html [hunk 7]
```diff
@@ -292,14 +306,101 @@ <h4 class="card__footer">27 Dec 2022,<span class="card__footer--gray"> by Roxie
...
+                        <li class="item item--mark">Company</li>
```
Comment (phattruong-agilityio) [inline practice-01/index.html]: wrong BEM name

File: practice-01/index.html [hunk 8]
```diff
@@ -292,14 +306,101 @@ <h4 class="card__footer">27 Dec 2022,<span class="card__footer--gray"> by Roxie
...
+                </div>
             </div>
-            <div class="footer__rules"></div>
+
         </div>
+
+        <hr class="footer__line">
+
+        <div class="footer__rule container">
+            <a href="mailto: abc@example.com " class="link-mail">© Copyright 2022 Yoora, Inc.</a>
```
Comment (phattruong-agilityio) [inline practice-01/index.html]: mail to ??

Comment (phattruong-agilityio) [reply practice-01/index.html]: Using HTML-encode for this character => `©`

File: practice-01/index.html [hunk 9]
```diff
@@ -292,14 +306,101 @@ <h4 class="card__footer">27 Dec 2022,<span class="card__footer--gray"> by Roxie
...
+
+            <div class="footer__rule-block">
+                <ul class="rules__list">
```
Comment (phattruong-agilityio) [inline practice-01/index.html]: Wrong BEM name

File: practice-01/index.html [hunk 10]
```diff
@@ -292,14 +306,101 @@ <h4 class="card__footer">27 Dec 2022,<span class="card__footer--gray"> by Roxie
...
+                    <li class="item">Terms of Service</li>
+                    <li class="item">Privacy Policy</li>
+                    <li class="item">Cookies</li>
+                </ul>
+                <div class="rules__button">
```
Comment (phattruong-agilityio) [inline practice-01/index.html]: rules__button ??

File: practice-01/index.html [hunk 11]
```diff
@@ -292,14 +306,101 @@ <h4 class="card__footer">27 Dec 2022,<span class="card__footer--gray"> by Roxie
...
+                    <button class="btn btn--group">
```
Comment (phattruong-agilityio) [inline practice-01/index.html]: btn--group is a bad name

File: practice-01/index.html (hunk 4 above)
Comment (TheHienDuong) [reply practice-01/index.html]: No need, because we can give the tag font--small nested

File: .vscode/settings.json [hunk 12]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (phattruong-agilityio) [inline .vscode/settings.json]: DONOT commit this file. It's out of project

File: practice-01/styles/font.css [hunk 13]
```diff
@@ -1,53 +1,49 @@
...
-    }
-    /* .h6-b3-text {
+/* Card */
+.card__text--lg {
+  font-size: var(--font-size-x-large);
+  font-weight: var(--font-weight-bold);
+  line-height: var(--line-height-xx-large);
+  letter-spacing: -0.33px;
+}
+.card__title {
```
Comment (phattruong-agilityio) [inline practice-01/styles/font.css:21]: break line here. Apply for all cases

File: practice-01/index.html [hunk 14]
```diff
@@ -290,16 +326,102 @@ <h4 class="card__footer">27 Dec 2022,<span class="card__footer--gray"> by Roxie
 
 
     <!-------------------
-    footer 
+    footer
```
Comment (phattruong-agilityio) [inline practice-01/index.html]: Lowercase?

File: practice-01/index.html [hunk 15]
```diff
@@ -290,16 +326,102 @@ <h4 class="card__footer">27 Dec 2022,<span class="card__footer--gray"> by Roxie
...
     --------------------->
-    <footer>
-        <div class="container">
-            <div class="footer__content">
-                <div class="content__left"></div>
-                <div class="content__right"></div>
+    <footer class="footer">
+        <div class="footer__content container">
+            <div class="content-left">
```
Comment (phattruong-agilityio) [inline practice-01/index.html]: Wrong BEM name. Update to all similar cases

File: practice-01/index.html [hunk 16]
```diff
@@ -290,16 +326,102 @@ <h4 class="card__footer">27 Dec 2022,<span class="card__footer--gray"> by Roxie
...
+            <a class="link-mail">&#169;Copyright 2022 Yoora, Inc.</a>
+
+            <div class="footer__rule-block">
+                <ul class="privacy-list">
+                    <li class="item">Terms of Service</li>
+                    <li class="item">Privacy Policy</li>
+                    <li class="item">Cookies</li>
+                </ul>
+                <div class="locale-switche">
+                    <button class="btn btn--switcher">
```
Comment (phattruong-agilityio) [inline practice-01/index.html]: btn--switcher => bad name. It should relative to css in this case. let try `btn--flex`

**Review by phattruong-agilityio — APPROVED** [review]

**Pull request merged**